code/modules/mob/mob_movement.dm

```diff
@@ -188,8 +190,6 @@
 					L.lying = 90
 					L.update_canmove()
 				playsound(L.loc, pick('sound/effects/bodyscrape-01.ogg', 'sound/effects/bodyscrape-02.ogg'), 20, 1, -4) //Crawling is VERY quiet
-				L.visible_message("<span class='danger'>[L] crawls forward!</span>", \
```

Comment:
shouldn't delete this but rather make it show up if L.status_flags & NEARCRIT no?